### DISCUSSION TITLE
Why does it matter if this template is "alive" or "dead"

### DISCUSSION THREAD

**Discussion by unluckychild** [ColorlibHQ/AdminLTE#4858] [General]
Im using this template as HTML for front end, and PHP as backend.
It is working fine. the only thing you can "update" are plugins. But without updating they are still working fine. So why do you want to update it so badly?

**Comment by gmolina75** [comment]
bootstrap 5 compliance

**Comment by jcvasquez07** [comment]
I've been using v3.0.5 with bootstrap 5 in a project already in production for a few months already (ever since the first bs5 alfa release). I had to make a few changes to make it work, mainly modals, tooltips and popups, which now use _data-bs-_ instead of _data-_ attributes and some new classes in forms and text alignment, but it wasn't difficult. All the plugins l use still work fine. Some plugins I updated to new versions just because there were new versions, not because of compatibility issues. I will update as soon as the stable release is out but I'm not expecting too much of a difference.
